src/tools/support_documentation_tools.py

```
@@ -0,0 +1,2298 @@
+                        "content": content,
+                        "file_path": str(file_path.absolute()),
+                        "source_id": source_id,
+                        "frontmatter_override": frontmatter_override if frontmatter_override else None,
+                    }
+                )
+
+            # Progress tracking callback
+            progress_callback = None
+            if show_progress:
+
+                def progress_callback(current: int, total: int):
```

Flake8 (7.2.0)

[error] redefinition of unused 'progress_callback' from line 215

(F811)